src/Tomcat/TomcatModel.ts

```diff
@@ -53,32 +52,25 @@ export class TomcatModel {
             server.jvmOptions = result.concat([Constants.BOOTSTRAP_FILE, '"$@"']);
             return;
         }
-        await new Promise((resolve: () => void): void => {
-            const lineReader: readline.ReadLine = readline.createInterface({
-                input: fse.createReadStream(server.jvmOptionFile),
-                crlfDelay: Infinity
-            });
-            lineReader.on('line', (line: string) => {
-                if (line.startsWith('-')) {
-                    Constants.JVM_DEFAULT_OPTIONS_KEYS.forEach((key: string) => {
-                        if (line.startsWith(key)) {
-                            return;
-                        }
-                    });
-                    result = result.concat(line);
-                }
-            });
-            lineReader.on('close', () => {
-                const tmpDirConfiguration: string = result.find((element: string) => {
-                    return element.indexOf(Constants.JAVA_IO_TEMP_DIR_KEY) >= 0;
-                });
-                if (!tmpDirConfiguration || tmpDirConfiguration.length <= 0) {
-                    result = result.concat(`${Constants.JAVA_IO_TEMP_DIR_KEY}=${path.join(catalinaBase, 'temp')}`);
+        const filterFunction: (para: string) => boolean = (para: string): boolean => {
```

Comment:
(para: string) => boolean = (para: string): boolean => 
what does this mean?

Reply:
filterFunction signature